### ISSUE TITLE
commands

### ISSUE THREAD

**Issue by Kanorto** [Impostor/Impostor#451]
Will you be making commands to the impostor server? Perhaps there are already commands?

For example, make an impostor "nickname" command or a role "nickname" "role" command.

**Comment by miniduikboot** [comment]
You can use the Plugin API to create commands: https://github.com/Impostor/Impostor/blob/master/docs/Writing-a-plugin.md . Unfortunately I don't know of an up-to-date plugin that adds commands.

Please don't post the same comment across multiple issues (#449), it makes it really hard to follow the discussion.
